### PR TITLE
linkedlist: added a testsuite and two functions for converting

### PR DESCRIPTION
**Pull request by christianbender**
I added two new functions:
```python
def convert_to_list(number : int) -> "Node":
def convert_to_str(l : "Node") -> str:
```
And I added a testsuite for the linked list and the full functionality of the project (uses unittest).

### CODE REVIEW

File: linkedlist/add_two_numbers.py [hunk 1]
```diff
@@ -38,16 +39,91 @@ def add_two_numbers(left:"Node", right:"Node")->"Node":
     return head.next
 
 
+def convert_to_list(number : int) -> "Node":
+    """
```
Comment (goswami-rahul) [inline linkedlist/add_two_numbers.py]: The `"Node"` should be just `Node` in the typing hint.

File: linkedlist/add_two_numbers.py [hunk 2]
```diff
@@ -38,16 +39,91 @@ def add_two_numbers(left:"Node", right:"Node")->"Node":
...
+            current = current.next
+            remainder = quotient % 10
+            quotient //= 10
+        current.next = Node(remainder)
+        return head.next
+    else:
+        print("number must be positive!")
+
+
+def convert_to_str(l : "Node") -> str:
```
Comment (goswami-rahul) [inline linkedlist/add_two_numbers.py]: Same applies here. `"Node"` change to `Node`.

**Comment by goswami-rahul** [comment]
Good work. I have left a minor change in review.

**Comment by christianbender** [comment]
@goswami-rahul  The orginal code contains "Node", too. That is why I wrote "Node". See here: https://github.com/christianbender/algorithms/blob/master/linkedlist/add_two_numbers.py
But I will change my code across your request.

**Comment by goswami-rahul** [comment]
Oh I didn't see that. Thanks for the changes.

**Comment by christianbender** [comment]
@goswami-rahul  No problem! Thanks for merging!

**Pull request merged**